models/neurons/aeif_cond_alpha_neuron.nestml

```diff
@@ -105,23 +106,16 @@ model aeif_cond_alpha_neuron:
         spike
 
     update:
-        if is_refractory:
+        if refr_t > 0 ms:
             # neuron is absolute refractory, do not evolve V_m
-            refr_t -= resolution()
+            integrate_odes(w, refr_t)
         else:
-            # neuron not refractory, so evolve all ODEs (including V_m)
-            integrate_odes()
+            # neuron not refractory
+            integrate_odes(V_m, w)
 
-    onCondition(V_m >= V_peak):
+    onCondition(V_m >= V_peak and refr_t <= 0 ms):
```

Comment:
Here and in all other neuron models with a similar construct, I'd swap the order of conditions: 
` onCondition(refr_t <= 0 ms and V_m >= V_peak)`